websites/D/Di.FM/presence.ts

```diff
@@ -16,6 +16,14 @@ presence.on("UpdateData", () => {
 		presenceData.state =
 			document.querySelectorAll(".track-name")[0].textContent;
 		presenceData.smallImageKey = "play";
+
+		const str = document.querySelector(
```

Comment:
`str` is a poor variable name

Reply:
Plus, it's not possible to get directly the `src` of the image from the query ? instead of using `outerHTML`...

Reply:
> Plus, it's not possible to get directly the `src` of the image from the query ? instead of using `outerHTML`...

Changed to directly get src